[NETBEANS-54] Module Review java.api.common

mirkoperillo opened the pull request:
fix issue #86

mirkoperillo committed:
[NETBEANS-54] Module Review java.api.common

fix issue #86

emilianbold commented on java.api.common/src/org/netbeans/modules/java/api/common/project/ui/customizer/vmo/gen/CommandLine.tokens:
```diff
@@ -1,3 +1,21 @@
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
```
This file seems to be generated by ANTLR and have very little info.

Please exclude it in nbbuild/build.xml ("rat" task).

mirkoperillo committed:
Update build.xml

excluded file java.api.common/src/org/netbeans/modules/java/api/common/project/ui/customizer/vmo/gen/CommandLine.tokens

emilianbold committed:
Merge branch 'master' into fix-86

emilianbold reviewed (changes requested):
Now that CommandLine.tokens is excluded, you should remove the header you just added.

mirkoperillo committed:
Update CommandLine.tokens

removed the license header

emilianbold committed:
Merge branch 'master' into fix-86
